frontend/test/unit/specs/components/v-content-report-form.spec.js

```diff
@@ -154,13 +154,18 @@ describe("VContentReportForm", () => {
     })
   })
 
-  it("should not send other report if description is short", async () => {
+  it("submit button on other form should only be enabled if input is longer than 20 characters", async () => {
     ReportService.sendReport = jest.fn()
 
     render(VContentReportForm, options)
     await fireEvent.click(getOtherInput())
 
-    const description = "less than 20 chars"
+    // The minimum length for report description is 20 characters. This is invalid
+    const description = "1234567890123456789"
     await fireEvent.update(getDescriptionTextarea(), description)
+    expect(await getReportButton()).toHaveAttribute("aria-disabled", "true")
```

Comment:
`get*` methods are not async, so await isn't necessary here. `@testing-library/jest-dom` also has a matcher for this case: https://github.com/testing-library/jest-dom#tobedisabled

```suggestion
    expect(getReportButton()).toBeDisabled()
```